ext/url/lib.deno_url.d.ts

```diff
@@ -262,13 +262,14 @@ declare class URLPattern {
   /**
    * Test if the given input matches the stored pattern.
    *
-   * The input can either be provided as a url string (with an optional base),
-   * or as individual components in the form of an object.
+   * The input can either be provided as an absolute url string,
+   * relative url string with a base url, or as individual components
```

Comment:
This is not completely accurate: a relative url with a base URL technically can be absolute as well. The previous wording was more generic

Reply:
Agree, although the base is not optional if provided URL is relative. Not sure what should the wording be here tbh.